modules/mysql/src/test/java/org/testcontainers/junit/mysql/SimpleMySQLTest.java

```diff
@@ -47,8 +45,7 @@ public class SimpleMySQLTest extends AbstractContainerDatabaseTest {
     @Test
     public void testSimple() throws SQLException {
         try (
-            MySQLContainer<?> mysql = new MySQLContainer<>(MySQLTestImages.MYSQL_57_IMAGE)
-                .withConfigurationOverride("somepath/mysql_conf_override")
+            MySQLContainer<?> mysql = new MySQLContainer<>(MySQLTestImages.MYSQL_56_IMAGE)
```

Comment:
```suggestion
            MySQLContainer<?> mysql = new MySQLContainer<>(MySQLTestImages.MYSQL_57_IMAGE)
```
We can keep the previous image here.